service/insights_v3.go

```diff
@@ -0,0 +1,24 @@
+package service
+
+import (
+	. "github.com/talbright/go-desk/types"
+	. "github.com/talbright/go-desk/resource"
+	"net/http"
+)
+
+type InsightsV3Service struct {
+	client *Client
+}
+
+func (s *InsightsV3Service) Report(query *InsightsV3Query) (*InsightsV3Report, *http.Response, error) {
```

Comment:
It may be passed but not actually utilized. I only say this after reading the docs ((http://dev.desk.com/API/insights3/#meta-show) which granted could be wrong or outdated:

> The general V2 response headers described here (http://dev.desk.com/API/using-the-api/#rate-limits) are not modified by or used to determine the Insights3 concurrency limit.